build.gradle.kts

```diff
@@ -67,6 +71,37 @@ dependencies {
     implementation("org.postgresql:postgresql:${Versions.postgres}")
     testImplementation("com.opentable.components:otj-pg-embedded:${Versions.postgresEmbedded}")
 
+    // Kafka
+    val excludeLog4j = fun ExternalModuleDependency.() {
+        exclude(group = "log4j")
+    }
+    implementation("org.apache.kafka:kafka_2.13:${Versions.kafka}", excludeLog4j)
+    implementation("io.confluent:kafka-avro-serializer:${Versions.confluent}", excludeLog4j)
+    implementation("io.confluent:kafka-schema-registry:${Versions.confluent}", excludeLog4j)
```

Comment:
Litt usikker på om vi trenger disse?

Reply:
Jeg har ikke lyst på avro, så for min del kan du bare fjerne det 👍

Reply:
Ja avro var litt for mye copy+pasting ser jeg. Vet ikke med schema registry, men det brukes jo i `ApplicationEnvironmentKafka` så mulig den må med

Reply:
Hmm den var ikke happy da jeg fjernet den igjen, men det kan jo være fordi det er et oppsett vi egentlig ikke ønsker å ta i bruker 🤷🏼‍♂️